xcell/mappers/mapper_SPT.py

```diff
@@ -23,12 +24,14 @@ def _get_hm_maps(self):
             hm1_map = hp.read_map(self.file_hm1)
             hm1_map[hm1_map == hp.UNSEEN] = 0.0
             hm1_map[np.isnan(hm1_map)] = 0.0
+            hm1_map = rotate_map(hm1_map, self.rot)
```

Comment:
Just a note: these maps are super high resolution (Nside=8192), so rotating them might be super slow. We should bear this in mind in case this strategy becomes a problem.